src/celeritas/optical/interactor/RayleighInteractor.hh

```diff
@@ -37,8 +37,8 @@ class RayleighInteractor
 {
   public:
     // Construct interactor from an optical track
-    inline CELER_FUNCTION
-    RayleighInteractor(TrackView const& particle, Real3 const& direction);
+    inline CELER_FUNCTION RayleighInteractor(ParticleTrackView const& particle,
+                                             Real3 const& direction);
```

Comment:
Clang-format differences 😒 really gotta get this into the CI, don't worry about it here